src/v3/src/types/schema.ts

```diff
@@ -300,12 +301,11 @@ export interface DescriptionElement extends UISchemaElement {
   };
 }
 
-export interface TextWithHtmlElement extends UISchemaElement {
-  type: 'TextWithHtml';
+export interface TextWithActionLinkElement extends UISchemaElement {
+  type: 'TextWithActionLink';
   options: ActionOptions & {
     content: string;
     contentClassname: string;
-    submitOnClick: boolean;
```

Comment:
Removed this because now the TextWithHtml component is renamed to TextWithActionLink. This was done to limit the usage of this component to only be used for translation keys that contain a link which does not contain an `href` value (i.e. it operates as a button instead of a link).